packages/harmony/src/components/button/OptionsFilterButton/OptionsFilterButton.tsx

```diff
@@ -129,7 +138,6 @@ export const OptionsFilterButton = forwardRef<
           transformOrigin={popupTransformOrigin}
           portalLocation={popupPortalLocation}
           zIndex={popupZIndex}
-          onAfterClose={() => setFilterInputValue('')}
```

Comment:
I think we still need this, because the filterInput should be empty every time you open the popup (even if you don't select anything)

Reply:
there was a bug before which i was trying to fix, but it turns out that both are needed to make it work correctly so there might be a better way to do this